.github/workflows/ci.yml

```diff
@@ -23,12 +23,20 @@ jobs:
           node-version: "12"
       - run: cp .env.sample .env
```

Comment:
Is the only reason we need an `.env` file for this environment for tests that include "Okta"?

Reply:
Like, do we need to run this `cp` command for an env vile to run a linter?

Reply:
I think these are leftovers from when we used to use a SONAR_TOKEN and GITHUB_TOKEN. You can see the tokens in the previous workflow on main. I agree. I will remove them.

Reply:
The app can't run if the env variables don't exist. I think it's mostly an issue with knex. For now we just need to use the dummy sample .env file.

Reply:
it's also the cheapest of all the run tasks in the job so doesn't "hurt" anything.